master/internal/command/events.go

```diff
@@ -135,15 +134,21 @@ func (e *eventManager) Receive(ctx *actor.Context) error {
 func canAccessCommandEvents(ctx *actor.Context, c echo.Context) error {
 	curUser := c.(*context.DetContext).MustGetUser()
 	taskID := model.TaskID(ctx.Self().Parent().Address().Local())
-	ownerID, err := db.GetCommandOwnerID(c.Request().Context(), taskID)
```

Comment:
this was assuming that the snapshot is always available. I'll check when we do this

Reply:
commands are persisted before they come online. we should be good to rely on this
https://github.com/hamidzr/determined/blob/040be76a417f4a1a5ff6626e15d199341a8a9540/master/internal/command/command.go#L328